guide/how-it-works/liquidity.md

```diff
@@ -0,0 +1,314 @@
+   alt-text = "Lori, Lamar, Lindsay, and the lightning network"
+   width = 1200
+   height = 800
+   layout = "full-width"
+%}
+
+Getting liquidity is not a one time issue on the lightning network. Routing nodes are constantly rebalancing their channels so that they have the right amount of liquidity in the right places.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/liquidity/liquidity-07.png"
+   retina = "/assets/images/guide/how-it-works/liquidity/liquidity-07@2x.png"
+   mobile = "/assets/images/guide/how-it-works/liquidity/liquidity-07-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/liquidity/liquidity-07-mobile@2x.png"
+   alt-text = "Rebalancing channels on the lightning network"
+   width = 1200
+   height = 800
+   layout = "full-width"
+%}
+
+While the example thus far has focused on routing nodes, mobile nodes also fit into this scenario. Now that our node operator Lori has maintained a well-connected routing node, she can use her node to extend liquidity to mobile nodes.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/liquidity/liquidity-08.png"
+   retina = "/assets/images/guide/how-it-works/liquidity/liquidity-08@2x.png"
+   mobile = "/assets/images/guide/how-it-works/liquidity/liquidity-08-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/liquidity/liquidity-08-mobile@2x.png"
+   alt-text = "Rebalancing channels on the lightning network"
+   width = 1200
+   height = 800
+   layout = "full-width"
+%}
+
+## Handling liquidity for wallet users
+
+The above examples were designed to help you understand liquidity on a conceptual level. Ideally, the user of a mobile [daily spending wallet]({{'/guide/daily-spending-wallet/' | relative_url}}) never has to consider any of the liquidity sourcing that happens behind the scenes. Consider how you might design a bitcoin product that doesn't require the user to understand lightning channel management.
+
+### Inbound liquidity
+
+Helping the user get inbound liquidity is probably more important than outbound liquidity for most use cases. Without inbound liquidity, the user can not receive bitcoin over lightning. This makes it very difficult for a new bitcoin user to receive their first bitcoin.
+
+If the user does not have bitcoin in their wallet, they do not need outbound liquidity. On the other hand, if the user has inbound liquidity, then they are free to receive bitcoin, and then end up with outbound liquidity from the bitcoin they received.
+
+One way to help your users receive inbound liquidity is through LSPs (Lightning Service Providers). You could run your own LSP service or integrate your product with an existing LSP.
+
+[Read the LSP section]({{'/guide/how-it-works/lightning-service-providers/#inbound-liquidity' | relative_url}}) to learn more. [Breez](https://breez.technology/) is an example of a wallet that does this.
+
+### Outbound liquidity
```

Comment:
Mentioned above, but you may want to state that channel opens usually only give the initiating party liquidity (that being outbound for them). 

You should add something in this bottom section on cooperative channel opens and how they can be used to open a channel with both parties having liquidity. 

Another thing you could mention as a TBC is channel splicing and how liquidity can be adjusted without channel opens / closes.

Reply:
Personally, I'd love to talk about dual-funded channels. But I'm writing this page for a designer audience and I'm trying to take a very complicated topic and distill it down into the minimum amount of information they need to understand lightning. I don't want to mention every single aspect of lightning technology, especially parts that aren't widely adopted yet. It's meant to be a general overview.

Reply:
Yeah I don't think we need to go in-depth but I don't see the harm in mentioning it's existence and maybe linking to external source if they want to go more in-depth

Reply:
Maybe some of those extras could be listed at the bottom of the page under "further reading" or "dive deeper"? That way they don't interrupt the flow.

Reply:
Added "further reading" tip box on dual funded channels and channel splicing immediately after the liquidity diagram section.